From eb71a518051e6ca4b514af185be9b4e115c72a93 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Tue, 1 Nov 2016 02:36:42 +0300
Subject: [PATCH 13/24] ARM64: dts: r8a7796-m3ulcb-view: Add more buttons to
 gpio keys

This adds SW4 and SW5 to gpio-keys device node,
and updates SW3 button name

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a7796-m3ulcb-view.dts | 16 +++++++++++++++-
 1 file changed, 15 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb-view.dts b/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb-view.dts
index 3235c35..24b0b6b 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb-view.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb-view.dts
@@ -85,13 +85,27 @@
 	keyboard {
 		compatible = "gpio-keys";
 
-		key-1 {
+		button@1 {
 			linux,code = <KEY_1>;
 			label = "SW3";
 			wakeup-source;
 			debounce-interval = <20>;
 			gpios = <&gpio6 11 GPIO_ACTIVE_LOW>;
 		};
+		button@2 {
+			linux,code = <KEY_2>;
+			label = "SW4";
+			wakeup-source;
+			debounce-interval = <20>;
+			gpios = <&gpio6 12 GPIO_ACTIVE_LOW>;
+		};
+		button@3 {
+			linux,code = <KEY_3>;
+			label = "SW5";
+			wakeup-source;
+			debounce-interval = <20>;
+			gpios = <&gpio6 13 GPIO_ACTIVE_LOW>;
+		};
 	};
 
 	leds {
-- 
1.9.3

